code/game/objects/effects/decals/remains.dm

```diff
@@ -39,7 +39,7 @@
 			COOLDOWN_START(src, gas_cooldown, rand(20 SECONDS, 2 MINUTES))
 
 /obj/effect/decal/remains/human/smokey/proc/release_smoke(mob/living/smoke_releaser)
-	visible_message(span_warning("[smoke_releaser] disturbs the [src], which releases a huge cloud of [that_shit_that_killed_saddam.name]!"))
+	visible_message(span_warning("[smoke_releaser] disturbs the [src], which releases a huge cloud of gas!"))
```

Comment:
worth noting you coulda used initial(that_shit_that_killed_sadam.name), it's how you access non list vars off typepaths